int32 vid
int32 vstate
---
bool status